CHANGELOG.rst

```diff
@@ -1,3 +1,10 @@
+1.3.1 (2024-02-29)
+------------------
+
+* Fix grouping under GitHub actions: print without colors, otherwise the ``::group::`` instructions are not processed.
+* When running under GitHub actions, print the ``::endgroup::``  prefix when finishing processing.
```

Comment:
Maybe the help (and/or docs) should be updated to mention this.

Reply:
Good point. :+1:

Reply:
Done, thanks.